_inc/client/privacy/index.jsx

```diff
@@ -129,8 +136,16 @@ class Privacy extends React.Component {
 export default connect(
 	state => ( {
 		settings: getSettings( state ),
+		tracking: getTrackingSettings( state ),
+		isUpdating: isUpdatingTrackingSettings( state ),
```

Comment:
Do we actually need `isUpdating` here? I can't find a place where we're using it.